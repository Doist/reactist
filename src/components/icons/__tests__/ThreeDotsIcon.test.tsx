import * as React from 'react'
import { render } from '@testing-library/react'

import ThreeDotsIcon from '../ThreeDotsIcon.svg'

describe('ThreeDotsIcon', () => {
    it('renders a svg icon', () => {
        const { container } = render(<ThreeDotsIcon />)
        expect(container).toMatchSnapshot()
    })
})
