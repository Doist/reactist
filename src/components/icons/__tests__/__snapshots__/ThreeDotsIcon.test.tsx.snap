// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreeDotsIcon renders a svg icon 1`] = `
<div>
  <svg
    height="26px"
    viewBox="0 0 26 26"
    width="26px"
  >
    <g
      fill="none"
      fill-rule="evenodd"
      stroke="none"
      stroke-width="1"
    >
      <circle
        cx="19"
        cy="13"
        fill="#A5A5A5"
        r="2"
      />
      <circle
        cx="13"
        cy="13"
        fill="#A5A5A5"
        r="2"
      />
      <circle
        cx="7"
        cy="13"
        fill="#A5A5A5"
        r="2"
      />
    </g>
  </svg>
</div>
`;
